WordPress/Docs/DB/RestrictedClassesStandard.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Restricted Database Classes and Methods"
+    >
+    <standard>
+    <![CDATA[
+      Avoid touching the database directly with PHP library classes and methods. Use the $wpdb object and associated functions instead.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Invalid: Using the object-orientated mysqli interface">
```

Comment:
All of the other code documentation files have the `valid` case first. Is there any reason this has the `invalid` cases first?

Reply:
I suspect that most remarks I left for #2453 will also apply to this sniff ?